buildSrc/win.gradle

```diff
@@ -452,3 +460,20 @@ WIN.webkit.linker = linker
 WIN.webkit.rcCompiler = rcCompiler
 WIN.webkit.rcSource = defaultRcSource
 WIN.webkit.rcFlags = ["/d", "JFX_FNAME=jfxwebkit.dll", "/d", "JFX_INTERNAL_NAME=webkit", rcFlags].flatten();
+
+String getWinArch(String arch) {
+    switch (arch) {
+        case "aarch64" : return "arm64"
+        case "amd64" : return "x64"
+        default: return arch
```

Comment:
This looks fine. I recommend to sanity test this on a 32-bit build to ensure that `os.arch` is `x86` (I don't have the ability to do that).

Reply:
Here are some println lines which I added to win.gradle file before building the fix for windows 32 bits
```
println "CPU_BITS: $CPU_BITS, ext.OS_ARCH: ${ext.OS_ARCH}"
println "HOST_ARCH: $HOST_ARCH, TARGET_ARCH: ${TARGET_ARCH}, IS_CROSS: ${IS_CROSS}"
```

I built the fix on Windows x86_64 with 32bits java
The output was:
```
CPU_BITS: x86, ext.OS_ARCH: x86
HOST_ARCH: x86, TARGET_ARCH: x86, IS_CROSS: false
```

`file` showed that the glass.dll has been build for Intel 80386:
```
file build/sdk/bin/glass.dll 
build/sdk/bin/glass.dll: PE32 executable (DLL) (GUI) Intel 80386, for MS Windows
```
I built corresponding jdk 16 with `--with-target-bits=32` option and the jfx and was able to successfully run Ensemble8.jar app on Windows 32bits.